config/deployer.sample.json

```diff
@@ -51,13 +51,28 @@
         "Name":  "Stack In Use",
         "Query": "avg(go_memstats_stack_inuse_bytes{instance=~\"app.*:8067\"})"
       },
+      {
```

Comment:
While we are here, I have seen that the CPU utilization graph is much smoother with rate rather than irate. Can we change that as part of this PR?

https://prometheus.io/docs/prometheus/latest/querying/functions/#irate

> irate should only be used when graphing volatile, fast-moving counters. Use rate for alerts and slow-moving counters, as brief changes in the rate can reset the FOR clause and graphs consisting entirely of rare spikes are hard to read.

Reply:
I think we are already using `irate`, or did you mean to use it for some other metric?

https://github.com/mattermost/mattermost-load-test-ng/blob/18a95305ec4e0cbe4bbde8b766be08c55f5870d2/config/deployer.sample.json#L43-L44

Reply:
I might have misread, were you proposing of switching from `irate` to `rate` to make the graph more readable?

Reply:
Correct.